kvs/client/python/anna/ipc_client.py

```diff
@@ -38,11 +38,9 @@ def __init__(self, thread_id = 0):
         self.put_request_socket.connect(PUT_REQUEST_ADDR)
 
         self.get_response_socket = self.context.socket(zmq.PULL)
-        self.get_response_socket.setsockopt(zmq.RCVTIMEO, 5000)
```

Comment:
Why are we dropping this timeout?

Reply:
the `kvs_async_client.hpp` is checking the timeout already and the `async_cache.cpp` is resending the requests if timed out (all these happen in the cache layer). But yeah I think it makes sense to check again here because the function executor may decide to do something other than resending the request. I'll add them back.